src/Tests/Microsoft.NET.Pack.Tests/GivenThatWeWantToPackAHelloWorldProject.cs

```diff
@@ -85,19 +88,21 @@ public void It_fails_if_nobuild_was_requested_but_build_was_invoked()
                 .HaveStdOutContaining("NETSDK1085");
         }
 
-        [Fact]
-        public void It_packs_with_release_if_PackRelease_property_set()
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void It_packs_with_release_if_PackRelease_property_set(bool optedOut)
         {
+            if (optedOut)
+            {
+                Environment.SetEnvironmentVariable(EnvironmentVariableNames.DISABLE_PUBLISH_AND_PACK_RELEASE, "true");
+            }
+
             var helloWorldAsset = _testAssetsManager
-               .CopyTestAsset("HelloWorld", "PackReleaseHelloWorld")
+               .CopyTestAsset("HelloWorld", $"PackReleaseBasicTest-Opted-Out-{optedOut}")
                .WithSource();
 
-            System.IO.File.WriteAllText(helloWorldAsset.Path + "/Directory.Build.props", "<Project><PropertyGroup><PackRelease>true</PackRelease></PropertyGroup></Project>");
-
-            new BuildCommand(helloWorldAsset)
-               .Execute()
-               .Should()
-               .Pass();
+            File.WriteAllText(helloWorldAsset.Path + "/Directory.Build.props", "<Project><PropertyGroup><PackRelease>true</PackRelease></PropertyGroup></Project>");
```

Comment:
Nit: Use `Path.Combine`

```suggestion
            File.WriteAllText(Path.Combine(helloWorldAsset.Path, "Directory.Build.props"), "<Project><PropertyGroup><PackRelease>true</PackRelease></PropertyGroup></Project>");
```